pom.xml

```diff
@@ -119,6 +119,8 @@
     <accumulo.release.version>${project.version}</accumulo.release.version>
     <!-- bouncycastle version for test dependencies -->
     <bouncycastle.version>1.50</bouncycastle.version>
+    <!-- Curator version -->
+    <curator.version>2.7.1</curator.version>
```

Comment:
Do we anticipate needing to control this on the command line, or elsewhere? We generally don't need properties for dependency versions (especially for test dependencies) unless the same version is used repeatedly for multiple artifacts (like jetty or slf4j) or if we anticipate swapping it out frequently during different build environments.